docs/customization.md

```diff
@@ -74,6 +74,96 @@ keys:
   }
   ```
 
+- **`"alertsOptions"`**\
+  Options to customize
+  [Alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)
+  beyond the 5 default ones used by GitHub. You may also know this feature by
+  the name of [Obsidian Callouts](https://help.obsidian.md/callouts).
+
+  ```json
+  "alertsOptions": {
+    "icons": {
+      "foo": "zap",
+      "bar": "./icons/flower.svg"
+    "titles": {
+      "bar": "Tip of the day"
+    },
+    "fallbackIcon": "alert"
+  }
+  ```
+
+  In more detail:
+
+  - **`"icons"`**
+
+    **Key:** Any custom marker name. Can also be a multi-word string. Has to be
+    fully lowercased.
+
+    **Value:** A custom svg to use in one of three formats:
+
+      1. Name of an [octicon](https://primer.style/octicons/) (in kebab-case)
+      2. Path to an svg file, either relative to Vivify's config directory, or
+      an absolute path. Tilde (`~`) can be used in an absolute path.
+      3. A raw svg tag as a string. Note: must escape double quotes or use
+      single quotes inside the string.
+
+    > [!WARNING]
+    > We're considering switching from octicons to lucide as the main icon
+    > provider, so option 1 is subject to change.
```

Comment:
Double checked if the markdown renders on GitHub because it has weird syntax hl on the diff

Ironically, GitHub can't render this GitHub Alert, but Vivify can :smile: